provider/ec2/image_test.go

```diff
@@ -46,13 +46,13 @@ var findInstanceSpecTests = []struct {
 		series: "xenial",
 		arches: []string{"amd64"},
 		cons:   "cores=4",
-		itype:  "c4.xlarge",
+		itype:  "t2.xlarge",
 		image:  "ami-00000133",
 	}, {
 		series: "xenial",
 		arches: []string{"amd64"},
 		cons:   "mem=10G",
-		itype:  "r3.large",
+		itype:  "r4.large",
```

Comment:
Theses updates look correct to me for the newly generated pricelist, but make me question the test. I guess we are still forcing ourselves to check the pricelist to make these pass.